Generated client fails flake8 due to trailing comma

**Describe the bug**
A trailing comma in the type signature of the endpoint_macros template causes Union types to include a trailing comma after the last type in the union.

**To Reproduce**
We run the following script to generate our client:
``` bash
#!/usr/bin/env bash
set -x  # added for clarity below
rm -rf test-api-client
openapi-python-client generate --path test.yaml
autoflake --in-place --remove-all-unused-imports --remove-unused-variables --expand-star-imports -r test-api-client/
isort test-api-client/
black test-api-client/
# Ignore lines too long and bare except
flake8 --ignore=E722,E501 test-api-client/
```

With 0.6.0-alpha release it produces the following output:
```
+ rm -rf test-api-client
+ openapi-python-client generate --path test.yaml
Generating test-api-client
+ autoflake --in-place --remove-all-unused-imports --remove-unused-variables --expand-star-imports -r test-api-client/
+ isort test-api-client/
+ black test-api-client/
reformatted /Users/daniel/openapi-specs1/test-api-client/test_api_client/models/__init__.py
All done! ✨ 🍰 ✨
1 file reformatted, 8 files left unchanged.
+ flake8 --ignore=E722,E501 test-api-client/
test-api-client/test_api_client/api/default/create_cat.py:33:36: E231 missing whitespace after ','
test-api-client/test_api_client/api/default/create_cat.py:43:36: E231 missing whitespace after ','
test-api-client/test_api_client/api/default/create_cat.py:56:36: E231 missing whitespace after ','
test-api-client/test_api_client/api/default/create_cat.py:73:36: E231 missing whitespace after ','
test-api-client/test_api_client/api/default/create_cat.py:86:36: E231 missing whitespace after ','
test-api-client/test_api_client/api/default/create_cat.py:102:36: E231 missing whitespace after ','
```

**Expected behavior**
Since flake8 is part of our pipeline, we'd love for it to pass on the generated client.

**OpenAPI Spec File**
https://gist.github.com/dtkav/3f4525af7aa7f95548c6fc2bac2eef1d

**Desktop (please complete the following information):**
 - OS: macOS 10.15.6
 - Python Version: 3.8.0
 - openapi-python-client version v.0.6.0-alpha.1

**Additional context**
I put up PR #178 which seems to fix the issue.

--- a/end_to_end_tests/golden-master/my_test_api_client/api/tests.py
+++ b/end_to_end_tests/golden-master/my_test_api_client/api/tests.py
@@ -14,9 +14,7 @@ from ..models.http_validation_error import HTTPValidationError
 
 def get_user_list(
     *, client: Client, an_enum_value: List[AnEnum], some_date: Union[datetime.date, datetime.datetime],
-) -> Union[
-    List[AModel], HTTPValidationError,
-]:
+) -> Union[List[AModel], HTTPValidationError]:
 
     """ Get a list of things  """
     url = "{}/tests/".format(client.base_url)
@@ -112,9 +110,7 @@ def get_basic_list_of_booleans(*, client: Client,) -> List[bool]:
 
 def upload_file_tests_upload_post(
     *, client: Client, multipart_data: BodyUploadFileTestsUploadPost, keep_alive: Optional[bool] = None,
-) -> Union[
-    None, HTTPValidationError,
-]:
+) -> Union[None, HTTPValidationError]:
 
     """ Upload a file  """
     url = "{}/tests/upload".format(client.base_url)
@@ -133,11 +129,7 @@ def upload_file_tests_upload_post(
         raise ApiResponseError(response=response)
 
 
-def json_body_tests_json_body_post(
-    *, client: Client, json_body: AModel,
-) -> Union[
-    None, HTTPValidationError,
-]:
+def json_body_tests_json_body_post(*, client: Client, json_body: AModel,) -> Union[None, HTTPValidationError]:
 
     """ Try sending a JSON body  """
     url = "{}/tests/json_body".format(client.base_url)
@@ -171,9 +163,7 @@ def test_defaults_tests_test_defaults_post(
     ),
     union_prop: Optional[Union[Optional[float], Optional[str]]] = "not a float",
     enum_prop: Optional[AnEnum] = None,
-) -> Union[
-    None, HTTPValidationError,
-]:
+) -> Union[None, HTTPValidationError]:
 
     """  """
     url = "{}/tests/test_defaults".format(client.base_url)

--- a/end_to_end_tests/golden-master/my_test_api_client/async_api/tests.py
+++ b/end_to_end_tests/golden-master/my_test_api_client/async_api/tests.py
@@ -14,9 +14,7 @@ from ..models.http_validation_error import HTTPValidationError
 
 async def get_user_list(
     *, client: Client, an_enum_value: List[AnEnum], some_date: Union[datetime.date, datetime.datetime],
-) -> Union[
-    List[AModel], HTTPValidationError,
-]:
+) -> Union[List[AModel], HTTPValidationError]:
 
     """ Get a list of things  """
     url = "{}/tests/".format(client.base_url,)
@@ -117,9 +115,7 @@ async def get_basic_list_of_booleans(*, client: Client,) -> List[bool]:
 
 async def upload_file_tests_upload_post(
     *, client: Client, multipart_data: BodyUploadFileTestsUploadPost, keep_alive: Optional[bool] = None,
-) -> Union[
-    None, HTTPValidationError,
-]:
+) -> Union[None, HTTPValidationError]:
 
     """ Upload a file  """
     url = "{}/tests/upload".format(client.base_url,)
@@ -139,11 +135,7 @@ async def upload_file_tests_upload_post(
         raise ApiResponseError(response=response)
 
 
-async def json_body_tests_json_body_post(
-    *, client: Client, json_body: AModel,
-) -> Union[
-    None, HTTPValidationError,
-]:
+async def json_body_tests_json_body_post(*, client: Client, json_body: AModel,) -> Union[None, HTTPValidationError]:
 
     """ Try sending a JSON body  """
     url = "{}/tests/json_body".format(client.base_url,)
@@ -178,9 +170,7 @@ async def test_defaults_tests_test_defaults_post(
     ),
     union_prop: Optional[Union[Optional[float], Optional[str]]] = "not a float",
     enum_prop: Optional[AnEnum] = None,
-) -> Union[
-    None, HTTPValidationError,
-]:
+) -> Union[None, HTTPValidationError]:
 
     """  """
     url = "{}/tests/test_defaults".format(client.base_url,)

--- a/openapi_python_client/templates/endpoint_macros.pyi
+++ b/openapi_python_client/templates/endpoint_macros.pyi
@@ -61,7 +61,7 @@ if {{ property.python_name }} is not None:
 {% else %}
 ) -> Union[
     {% for response in endpoint.responses %}
-    {{ response.return_string() }},
+    {{ response.return_string() }}{{ "," if not loop.last }}
     {% endfor %}
 ]:
 {% endif %}

